app/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -1404,9 +1959,9 @@ def test_nimbus_config(self):
                         description
                         isDefault
                         metrics {
-                        slug
-                        friendlyName
-                        description
+                            slug
```

Comment:
I wonder with these changes as auto format should have done this before

Reply:
Unfortunately not, it just treats the gql query as a string so it doens't get autoformatted at all 😞